.gitignore

```diff
@@ -0,0 +1,2 @@
+
+mongoose-free-6.9.exe
```

Comment:
let's not commit this file since it's not common to everyone on the project. maybe add a line to `.git/info/exclude` or `~/.gitignore_global` instead? see here for more info: https://stackoverflow.com/questions/10176875/add-gitignore-to-gitignore/10177000